Format Python script using Black formatter

za opened the pull request:
<!-- Thank you for submitting a pull request to the WrongSecrets app! See what makes a good PR at https://github.com/OWASP/wrongsecrets/blob/master/CONTRIBUTING.md-->

### What kind of changes does this PR include?

-   [ ] Fixes or refactors
-   [ ] A new challenge
-   [ ] Additional documentation
-   [x] Something else

#### Description

<!---
Please provide a helpful summary of what change this pull request will introduce.
--->

Format Python script using Black formatter so that the style is consistent. Hopefully this will make the code more readable and attract more contributors.


### Relations

<!---
If your pull request fully resolves and should automatically close the linked issue, use Closes. Otherwise, use Relates.

For Example:

Relates #0000
or
Closes #0000
--->

Closes #1200 


### References

<!---
Optionally, provide any helpful references that may help the reviewer(s).
--->

https://github.com/psf/black

### Checklist:

-   [x] All the contributions made are solely the work of me and my co-authors
-   [x] I tested the changes in this PR (if applicable)
-   [x] I added unit tests to ensure my change works (when change in Java or on front-end code)
-   [x] I added UI tests to ensure my UI changes work (when change in the overall UI, not needed if just adding a challenge)
-   [x] The PR passes pre-commit hooks and automated tests

commjoen reviewed (changes requested):
Hi @za ,
Can you please take a look on how we can have the formatting done automatically using precommit? Thanks!

za commented:
Hi @commjoen ok, I'll have a look.

za commented:
> Hi @za , Can you please take a look on how we can have the formatting done automatically using precommit? Thanks!

Hi @commjoen I've added `black.yml` and updated `pre-commit.yml` file. Can you check?

commjoen commented on .github/workflows/pre-commit.yml:
```diff
@@ -23,6 +23,12 @@ jobs:
+        with:
+          options: "--check --verbose"
+          src: "./scripts"
+          version: "~= 22.0"
```
Hi @za ,
Can you remove this and update the precommit config instead please? This is  the GitHub workflow.

commjoen replied:
See https://github.com/OWASP/wrongsecrets/blob/master/.pre-commit-config.yaml

za replied:
> Hi @za , Can you remove this and update the precommit config instead please? This is the GitHub workflow.

Removed.

commjoen commented:
Something seems to fail when using the new pre-commit hook:

```sh
[26/01 4:42:29] ➜  sort_contibutors git:(python-black-formatted) ✗ pre-commit install
pre-commit installed at .git/hooks/pre-commit
[26/01 4:42:34] ➜  sort_contibutors git:(python-black-formatted) ✗ git commit        
[INFO] Initializing environment for https://github.com/psf/black.
An unexpected error has occurred: CalledProcessError: command: ('/usr/local/opt/git/libexec/git-core/git', 'checkout', '22.6b0')
return code: 1
expected return code: 0
stdout: (none)
stderr:
    error: pathspec '22.6b0' did not match any file(s) known to git
    
Check the log at /Users/razr/.cache/pre-commit/pre-commit.log
[26/01 4:42:38] ➜  sort_contibutors git:(python-black-formatted) ✗ git status
On branch python-black-formatted
Your branch is up to date with 'origin/python-black-formatted'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   main.py

```

za commented:
> Something seems to fail when using the new pre-commit hook, can you please fix the black integration :) ? Then I am very happy to continue :) .
> 
> ```shell
> [26/01 4:42:29] ➜  sort_contibutors git:(python-black-formatted) ✗ pre-commit install
> pre-commit installed at .git/hooks/pre-commit
> [26/01 4:42:34] ➜  sort_contibutors git:(python-black-formatted) ✗ git commit        
> [INFO] Initializing environment for https://github.com/psf/black.
> An unexpected error has occurred: CalledProcessError: command: ('/usr/local/opt/git/libexec/git-core/git', 'checkout', '22.6b0')
> return code: 1
> expected return code: 0
> stdout: (none)
> stderr:
>     error: pathspec '22.6b0' did not match any file(s) known to git
>     
> Check the log at /Users/razr/.cache/pre-commit/pre-commit.log
> [26/01 4:42:38] ➜  sort_contibutors git:(python-black-formatted) ✗ git status
> On branch python-black-formatted
> Your branch is up to date with 'origin/python-black-formatted'.
> 
> Changes to be committed:
>   (use "git restore --staged <file>..." to unstage)
> 	modified:   main.py
> ```

Updated in the next commit

commjoen commented on .github/workflows/black.yml:
```diff
@@ -0,0 +1,10 @@
+name: Black
+
+on: [push, pull_request]
```
can you please remove this file :) ?

za replied:
> can you please remove this file :) ? After that we should be good to merge and complete!

Removed.

commjoen reviewed (approved):
Thank you!
